base/applications/rapps/appview.cpp

```diff
@@ -108,25 +102,26 @@ VOID CMainToolbar::OnGetDispInfo(LPTOOLTIPTEXT lpttt)
 
 HWND CMainToolbar::Create(HWND hwndParent)
 {
+    /* Load tooltip strings */
+    szInstallBtn.LoadString(hInst, IDS_TOOLTIP_INSTALL);
+    szUninstallBtn.LoadString(hInst, IDS_TOOLTIP_UNINSTALL);
+    szModifyBtn.LoadString(hInst, IDS_TOOLTIP_MODIFY);
+    szSelectAll.LoadString(hInst, IDS_TOOLTIP_SELECT_ALL);
+    szRefreshBtn.LoadString(hInst, IDS_TOOLTIP_REFRESH);
+    szUpdateDbBtn.LoadString(hInst, IDS_TOOLTIP_UPDATE_DB);
+
     /* Create buttons */
     TBBUTTON Buttons[] =
     {   /* iBitmap, idCommand, fsState, fsStyle, bReserved[2], dwData, iString */
-        {  0, ID_TOOLBAR_INSTALL,   TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szInstallBtn      },
-        {  1, ID_UNINSTALL, TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szUninstallBtn    },
-        {  2, ID_MODIFY,    TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szModifyBtn       },
-        {  3, ID_CHECK_ALL, TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szSelectAll       },
+        {  0, ID_TOOLBAR_INSTALL,   TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szInstallBtn.GetString()      },
+        {  1, ID_UNINSTALL, TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szUninstallBtn.GetString()    },
+        {  2, ID_MODIFY,    TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szModifyBtn.GetString()       },
+        {  3, ID_CHECK_ALL, TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szSelectAll.GetString()       },
         { -1, 0,            TBSTATE_ENABLED, BTNS_SEP,                    { 0 }, 0, 0                           },
-        {  4, ID_REFRESH,   TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szRefreshBtn       },
-        {  5, ID_RESETDB,   TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szUpdateDbBtn      }
+        {  4, ID_REFRESH,   TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szRefreshBtn.GetString()       },
+        {  5, ID_RESETDB,   TBSTATE_ENABLED, BTNS_BUTTON | BTNS_AUTOSIZE, { 0 }, 0, (INT_PTR)szUpdateDbBtn.GetString()      }
     };
```

Comment:
since the indentation of this entire block from line 113-122 is not aligned properly anymore anyway I would recommend to entirely delete the space padding on each line before the final curly brace on each line. The closing curly baces aren't aligned at all. So it is better to do:
.GetString()       }
->
.GetString() }

Reply:
OK.